### PR TITLE
command search + project-id filter

### PR DESCRIPTION
**Pull request by grosser**
- search for text inside of commands, for example deprecated calls
 - filter by project to clean it out

/cc @zendesk/samson

screen shot 2016-03-01 at 10 04 15 pm
 

### Risks
- None

### CODE REVIEW

**Comment by grosser** [comment]
@zendesk/runway

**Comment by sandlerr** [comment]
are there too many commands to do a fuzzy search in JS?

**Comment by sandlerr** [comment]
perhaps something like https://github.com/atom/fuzzaldrin

**Comment by grosser** [comment]
we have 1500 ... don't want to load all this onto the page ... if we do fuzzy search then I'd prefer it on the backend

File: app/views/admin/commands/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,20 @@
 <h1>Commands</h1>
 
+<%= form_tag '?', method: :get, class: 'clearfix' do %>
+  <div class="col-sm-6" style="padding-left: 0;">
+    <%= label_tag "Search" %>
+    <%= text_field_tag 'search[query]', params[:search].try(:[], :query), class: "form-control" %>
+  </div>
+  <div class="col-sm-3">
+    <%= label_tag "Project" %><br/>
+    <% options = options_for_select(Project.pluck(:name, :id), params[:search].try(:[], :project_id)) %>
```
Comment (sandlerr) [inline app/views/admin/commands/index.html.erb]: we should go to Ruby 2.3, then you can `params.dig(:search, :project_id)`

**Comment by sandlerr** [comment]
:+1: as is, we can add fuzzy later

**Pull request merged**